pkg/server/smb/server.go

```diff
@@ -25,6 +25,20 @@ func normalizeWindowsPath(path string) string {
 	return normalizedPath
 }
 
+func normalizeMappingPath(path string) string {
+	items := strings.Split(path, "\\")
```

Comment:
is it possible that `path` is like `//hostname/share` ? do we need to handle such case?

Reply:
I will check